common/src/python/inputs/parameter_store.py

```diff
@@ -141,6 +141,30 @@ def get_api_key(self, path_prefix: str) -> str:
 
         return apikey
 
+    def get_redcap_parameters(self, *, base_path: str,
```

Comment:
Given that we are potentially impacting all projects, the pattern should be to get *all* of the redcap project parameters.

We might want to reconsider the parameter paths in SSM for these to avoid pulling in more than is necessary